tutorials/flume-capture-events.md

```diff
@@ -0,0 +1,199 @@
+---
+layout: page
+title: Capturing Events with Flume
+---
+
+## Purpose
+
+This lesson demonstrates how you can configure Flume to capture events from a web application with minimal impact on performance or the user. Flume collects individual events and writes them in groups to the dataset.
+
+The Flume agent receives the events over inter-process communication (IPC), and writes the events to the Hive file sink. Each time you send a message, Log4j writes a new `INFO` line in the terminal window.
```

Comment:
I don't think this needs to mention IPC. IPC covers a wide variety of communication methods. What actually happens is the Flume agent listens for events that are sent from an application, in this case the sample web application. That application sends events by "logging" them through Log4j, which also logs the event to the terminal.